docs/custom-design.rst

```diff
@@ -0,0 +1,53 @@
+Custom Design
+=============
+
+If you want to change the styles of the search UI,
+you can do so by adding your custom stylesheet to your documentation.
+You can read more about this on the official documentation
+of `Read the Docs`_.
```

Comment:
I suppose that "adding your custom stylesheet" should be clickable instead of Read the Docs.

Reply:
Yeah.. that sounds more readable.